source/puppet/5.3/hiera_automatic.md

```diff
@@ -29,57 +29,60 @@ For example, you can set servers for the `NTP` class like this:
 
-1. Global hierarchy.
-2. The current environment’s hierarchy.
-3. The indicated module’s hierarchy, if the key is of the form `<MODULE NAME>::<SOMETHING>`.
-4. If not found and the module's hierarchy has a default_hierarchy entry in its hiera.yaml - the lookup is repeated if steps 1-3 did not produce a value.
+1.  Global hierarchy.
+2.  The current environment's hierarchy.
+3.  The indicated module's hierarchy, if the key is of the form `<MODULE NAME>::<SOMETHING>`.
+4.  If not found and the module's hierarchy has a default_hierarchy entry in its hiera.yaml - the lookup is repeated if steps 1-3 did not produce a value.
 
 > Note: Hiera checks the global layer before the environment layer. If no global `hiera.yaml` file has been configured, Hiera defaults are used. If you do not want it to use the defaults, you can create an empty `hiera.yaml` file in `/etc/puppetlabs/puppet/hiera.yaml`.
 
 Default global hiera.yaml is installed at /etc/puppetlabs/puppet/hiera.yaml.
 
-### Arguments  
- 
+### Arguments
+
 You must provide the name of a key to look up, and can optionally provide other arguments. You can combine these arguments in the following ways:
-* `lookup( <NAME>, [<VALUE TYPE>], [<MERGE BEHAVIOR>], [<DEFAULT VALUE>] )`
-* `lookup( [<NAME>], <OPTIONS HASH> )`
-* `lookup( as above ) |$key| { <VALUE> } # lambda returns a default value`
- 
+-   `lookup( <NAME>, [<VALUE TYPE>], [<MERGE BEHAVIOR>], [<DEFAULT VALUE>] )`
+-   `lookup( [<NAME>], <OPTIONS HASH> )`
+-   `lookup( as above ) |$key| { <VALUE> } # lambda returns a default value`
+
 Arguments in `[square brackets]` are optional.
- 
-> Note: Giving a hash of options containing `default_value` at the same time as giving a lambda means that the lambda will win. The rationale for allowing this is that you may be using the same hash of options multiple times, and you may want to override the production of the default value. A `default_values_hash` wins over the lambda if it has a value for the looked up key. 
 
-### Merge behaviours   
- 
+> Note: Giving a hash of options containing `default_value` at the same time as giving a lambda means that the lambda will win. The rationale for allowing this is that you may be using the same hash of options multiple times, and you may want to override the production of the default value. A `default_values_hash` wins over the lambda if it has a value for the looked up key.
+
+### Merge behaviours
+
 Hiera uses a hierarchy of data sources, and a given key can have values in multiple sources. By default, Hiera returns the first value it finds, but it also continues to search and merge all the values together.
- 
+
 > Note: Data sources can use the `lookup_options` metadata key to request a specific merge behavior for a key. The lookup function will use that requested behavior unless you specify one.
 
 ### Examples
 
 Look up a key and returning the first value found:
 
-`lookup('ntp::service_name')`
```

Comment:
Even though this is one line of code, for consistency (especially for DITA migration), I put it in a code block.